src/main/java/com/bina/varsim/tools/simulation/RandVCF2VCF.java

```diff
@@ -45,13 +47,17 @@
     int numMNP = NUM_MNP_ARG;
     @Option(name = "-num_complex", usage = "Number of complex variants (other ones) to sample [" + NUM_COMPLEX_ARG + "]")
     int numComplex = NUM_COMPLEX_ARG;
+    @Option(name = "-num_dup", usage = "Number of complex variants (other ones) to sample [" + NUM_DUP_ARG + "]")
```

Comment:
So, you are reusing the ins/del params for short variants? If you are including indel SVs with the small SVs, the random sampling will most-likely end up having no indel SVs since they are very few in number.

Reply:
That's a good point. Here I adopted a workaround, please refer to https://github.com/bioinform/varsim/pull/149/files#diff-163a888df028d8bfa31a54c7682b7a12R112

basically, the `dgv_vcf` and `small_vcf` will be sampled independent of each other.